migrations/826-files-delete-unused-columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `files`
```

Comment:
This file needs to be renamed to `827-files-delete-unused-columns.sql` now that there's a `826...` migration on master.